assets/js/main.js

```diff
@@ -5,10 +5,19 @@ function displayViewport(event) {
   document.getElementById('h').innerHTML = verge.viewportH();
 }
 
+function explanation() {
+  var x = document.getElementById("Explination");
+  if (x.innerHTML == "") {
+    x.innerHTML = "Screen resolution is not the same as viewport size. Viewport refers to the visible area of a webpage on a device, while Screen resolution is the amount of pixels a device has";
```

Comment:
Instead of adding and removing content to the DOM with JS, can you instead just put this text inside the `div` and toggle its `display` value between `block`/`none`?